server/member/leader.go

```diff
@@ -330,6 +331,41 @@ func (m *Member) GetMemberLeaderPriority(id uint64) (int, error) {
 	return int(priority), nil
 }
 
+func (m *Member) getMemberBinaryDeployPath(id uint64) string {
+	return path.Join(m.rootPath, fmt.Sprintf("member/%d/deploy_path", id))
+}
+
+// GetMemberDeployPath loads a member's binary deploy path.
+func (m *Member) GetMemberDeployPath(id uint64) (string, error) {
+	key := m.getMemberBinaryDeployPath(id)
+	res, err := etcdutil.EtcdKVGet(m.client, key)
+	if err != nil {
+		return "", err
+	}
+	if len(res.Kvs) == 0 {
+		return "", errors.New("no value")
+	}
+	return string(res.Kvs[0].Value), nil
+}
+
+// SetMemberDeployPath saves a member's binary deploy path.
+func (m *Member) SetMemberDeployPath(id uint64) error {
+	key := m.getMemberBinaryDeployPath(id)
+	txn := kv.NewSlowLogTxn(m.client)
+	deployPath, err := os.Executable()
```

Comment:
need `filepath.Dir` to get the executable's dir path?

Reply:
Oh, you're right.